### PR TITLE
Add comments in htstress

### PR DESCRIPTION
**Pull request by Julian-Chu**
add comments in htstress.c

### CODE REVIEW

File: htstress.c [hunk 1]
```diff
@@ -65,6 +65,13 @@ typedef void (*sighandler_t)(int);
 
 #define MAX_EVENTS 256
 
+/**
+ * struct econn - represents one connection to server from htstress.
+ * @fd: clientfd
+ * @offs: the buffer offset represents data already read from or sent to buffer.
+ * @flags: flags represents the http response status from server, only used with
+ * BAD_REQUEST.
```
Comment (jserv) [inline htstress.c:73]: Indent this line. It should be aligned with "flags ".

File: htstress.c [hunk 2]
```diff
@@ -140,10 +147,16 @@ static void init_conn(int efd, struct econn *ec)
         exit(1);
     }
 
+    // manipulate fd, F_SETFL: set file status flags,
```
Comment (jserv) [inline htstress.c:150]: Always use C-style comments. That is, `/*` and `*/`.

File: htstress.c (hunk 1 above)
Comment (Julian-Chu) [reply htstress.c]: fixed in https://github.com/sysprog21/khttpd/pull/8/commits/be11e24a0e094e21f849d5e28b393a9f9df55cfc

File: htstress.c (hunk 2 above)
Comment (Julian-Chu) [reply htstress.c]: fixed in https://github.com/sysprog21/khttpd/pull/8/commits/be11e24a0e094e21f849d5e28b393a9f9df55cfc

**Pull request merged**